### PR TITLE
Adjusts one highly conflict-prone code snippet

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
This has caused so many conflicts it's painful

## Why It's Good For The Game
Much less conflicts whenever there's a species change

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -100,7 +100,40 @@
...
+	//if you don't keep this alphabetised I'm going to personally steal your shins and sell them online
+	return list(
+		"body_markings" = pick(GLOB.body_markings_list),
+		"body_size" = pick(GLOB.body_sizes),
+		"caps" = pick(GLOB.caps_list),
```
Comment (MarkSuckerberg) [inline code/__HELPERS/mobs.dm:107]: also what even ARE these they aren't used anywhere. might remove them

Comment (MarkSuckerberg) [reply code/__HELPERS/mobs.dm:107]: mushroom people, apparently

**Comment by triplezeta** [comment]
really good pr i think

**Comment by Fikou** [comment]
God bless

**Pull request merged**